src/GuiBase/Viewer/Gizmo/RotateGizmo.cpp

```diff
@@ -19,13 +19,19 @@
 
 namespace Ra {
 namespace Gui {
+
+inline void colorMesh( std::shared_ptr<Engine::Mesh> mesh, const Core::Color& color ) {
```

Comment:
Why is this function local to ``RotateGizmo.cpp``. The same is define also in ScaleGizmo.cpp and ``TranslateGizmo.cpp`` and ...
This method must be define only once, e.g. in ``Engine/Renderer/Mesh/Mesh.hpp`` as a static inline public method.